app/controllers/permissions_controller.rb

```diff
@@ -77,7 +77,23 @@ def edit
       add_breadcrumb @permission.fetch('catalog_item_identity', {})['name'], permission_path(@permission_concept_id)
       add_breadcrumb 'Edit', edit_permission_path(@permission_concept_id)
 
-      hydrate_groups(@permission)
+      # Searching for permission to get revision_id
+      @opts = {
+        'page_size'        => RESULTS_PER_PAGE,
+        'id'               => @permission_concept_id,
+        'include_full_acl' => true
+      }
+
+      permission_search_response = cmr_client.get_permissions(@opts, token)
+      permission_found = permission_search_response.body.fetch("hits", 0) > 0
```

Comment:
very minor - since there's no interpolation, `'hits'` should be in single quotes (rubocop should have said something)